jekyll/_cci2/getting-started.md

```diff
@@ -15,131 +15,190 @@ This document provides a step-by-step tutorial for getting your first successful
+## Digging Into Your First Pipeline
 
-3. To add your new repo, ensure that your GitHub account is selected in the dropdown in the upper-left, Select the Add Projects page, and find the repository you just created in the list (you can also search for it), then click the **Set Up Project** button next to it. ![]( {{ site.baseurl }}/assets/img/docs/CircleCI-add-new-project-list.png)
+You should see your pipeline start to run automatically—and pass! So, what just
+happened? Click on the green `Success` button on your pipeline to investigate
+the following parts of the run: 
 
-4. On the following screen, choose a language from the dropdown to get a pre-populated config.yml file with suggested best-practices for your project. If the language you are using is not in the list, click the "Use Hello World" button at the bottom of the list. Copy the YAML configuration that appears and save it as a file called "config.yml" in a folder named `.circleci` in the root of your repository's directory. ![]( {{ site.baseurl }}/assets/img/docs/CircleCI-add-new-project-list-2.png)
+![First Successful Pipeline]( {{ site.baseurl }}/assets/img/docs/getting-started--first-success.png){:.img--bordered}
 
-5. Git push your new  `config.yml` file to your VCS. You are ready to start building! Click the button "Start Building" and confirm in the modal that your repository is ready to build.
+1. **What workflows ran?**: After clicking "Success", we are taken to a page
+   listing the jobs that ran. If this is your first build, you probably only ran
+   **one job**  (which automatically runs inside **one workflow**).  In our
+   case, we only ran one job, called `Build`. Click on `Build` and let's
+   investigate the steps of our job. 
 
-## Running Your First CircleCI Build!
+  ![Investigate build]( {{ site.baseurl }}/assets/img/docs/getting-started--first-success-workflow.png){:.img--bordered}
 
-You should see your build start to run automatically—and pass! So, what just happened? Click on the green Success button on the CircleCI dashboard to investigate the following parts of the run:
 
-1. **Spin up environment:** CircleCI used the `circleci/ruby:2.4.1` Docker image to launch a virtual computing environment.
+1. **Spin up environment:** CircleCI used an [orb](https://circleci.com/orbs) to
+   help provide some defaults for this project. By using an orb, we can get
+   quick access to common configuration. In this case,
+   `circleci/welcome-orb@0.4.1` provides a "pre-built" job you can run which
+   simply greets the user. 
 
-2. **Checkout code:** CircleCI checked out your GitHub repository and "cloned" it into the virtual environment launched in Step 1.
+1. **Views step results:** Every job is made up of a series of steps - some
+   steps, like
+   [`checkout`]({{site.baseurl}}/2.0/configuration-reference/#checkout) are
+   built-in commands in CircleCI. Other steps are specified by a user to achieve
```

Comment:
In the config ref `checkout` and others that do magic fancy stuff are referred to as 'special steps', maybe that rather that build-in? I don't know...